model/src/main/proto/performance_metrics.proto

```diff
@@ -26,45 +25,78 @@ message OppiaMetricLog {
+      StorageUsage storage_usage = 2;
+
+      // The metric being logged is related to the startup latency of the application.
+      StartupLatency startup_latency = 3;
+
+      // The metric being logged is related to the memory usage of the application.
+      MemoryUsage memory_usage = 4;
+
+      // The metric being logged is related to the network usage of the application.
+      NetworkUsage network_usage = 5;
+
+      // The metric being logged is related to the cpu usage of the application.
+      CpuUsage cpu_usage = 6;
+    }
+  }
+
+  // Structure of the apk size metric.
   message ApkSize {
-    // Size of the android package file of the app on the device in bytes.
+    // Size of the app's installed APK file, in bytes.
     int64 apk_size_bytes = 1;
   }
 
-  // Structure of the storage usage package.
+  // Structure of the storage usage metric.
   message StorageUsage {
-    // Amount of storage usage by the app on user's device in bytes.
+    // Amount of storage usage by the app on user's device in bytes. This storage size is the cumulative
+    // size of app-specific files which include the application cache but not the apk size.
     int64 storage_usage_bytes = 1;
   }
 
-  // Structure of the startup latency package.
+  // Structure of the startup latency metric.
   message StartupLatency {
-    // Amount of latency on application startup in millis.
+    // Number of milliseconds required to start up the application from a cold start.
     int64 startup_latency_millis = 1;
   }
 
-  // Structure of the memory usage package.
+  // Structure of the memory usage metric.
   message MemoryUsage {
     // Amount of memory used by the application on the device in bytes.
+    // Here, PSS refers to Proportional Set Size which is the number of non-shared pages used by the
+    // app. It is majorly useful when we want to know how much memory is used by all processes since
+    // processes don't get counted multiple times. For more information, refer to the following link:
+    // https://developer.android.com/topic/performance/memory-management#calculating_memory_footprint
     int64 total_pss_bytes = 1;
   }
 
-  // Structure of the network usage package.
-  message NetworkUsage{
-    // Number of bytes received by the application on the device during the bucket's time interval.
+  // Structure of the network usage metric.
+  message NetworkUsage {
+    // Usage data is collected in discrete bins of time called 'Buckets'. This fields denotes the
+    // number of bytes received by the application on the device during this bucket's time interval.
     int64 bytes_received = 1;
 
-    // Number of bytes transmitted by the application from the device during the bucket's time
+    // Usage data is collected in discrete bins of time called 'Buckets'. This fields denotes the
+    // number of bytes transmitted by the application on the device during this bucket's time
     // interval.
     int64 bytes_transmitted = 2;
```

Comment:
Transmission can mean both sending and receiving. Perhaps be explicit here & in the documentation by indicating 'sent' (to contrast with 'received').

Reply:
done.